hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -28,10 +28,14 @@ $OVS_BR_ADAPTER = "br-int"
 $AntreaHnsNetworkName = "antrea-hnsnetwork"
 
 function RemoveOVSService() {
-    stop-service ovs-vswitchd
-    sc.exe delete ovs-vswitchd
-    stop-service ovsdb-server
-    sc.exe delete ovsdb-server
+    if (Get-Service ovs-vswitchd -ErrorAction SilentlyContinue) {
+        stop-service ovs-vswitchd
+        sc.exe delete ovs-vswitchd
+    }
+    if (Get-Service ovsdb-server -ErrorAction SilentlyContinue) {
+        stop-service ovsdb-server
+        sc.exe delete ovsdb-server
+    }
```

Comment:
Add documentation to say we need to set parameter "-RemoveOVS $false" when calling Clean-AntreaNetwork.ps1 if OVS is expected to run inside containers.

Reply:
Think about how to delete configurations from OVSDB when OVS userspace is running inside containers.

Reply:
> Add documentation to say we need to set parameter "-RemoveOVS $false" when calling Clean-AntreaNetwork.ps1 if OVS is expected to run inside containers.

It is false by default, so we don't need to set it explicitly. Even if it is true it removes ovs services if running on host. So, whether its true or false it doesn't affect if OVS is expected to run inside containers.

Reply:
We should avoid running ResetOVSService function because there is no ovs service installed on host.

Reply:
> > Add documentation to say we need to set parameter "-RemoveOVS $false" when calling Clean-AntreaNetwork.ps1 if OVS is expected to run inside containers.
> 
> It is false by default, so we don't need to set it explicitly. Even if it is true it removes ovs services if running on host. So, whether its true or false it doesn't affect if OVS is expected to run inside containers.

Clean-AntreaNetwork.ps1 is supposed to remove all ovs bridges to reset environment. In ovs container scenario, if we don't remove conf.db, ovs container will restore existing bridge when it restart, which means we can never clean the ovs bridge in this case. Uninstall-ovs.ps1 should not be used to reset host network.

Reply:
Reminder